projects/lib/src/tournament.h

```diff
@@ -196,12 +196,19 @@ class LIB_EXPORT Tournament : public QObject
 		void setEpdOutput(const QString& fileName);
 
 		/*!
-		 * Sets the opening repetition mode to \a repeat.
+		 * Sets the number of opening repetitions to \a count.
 		 *
-		 * If \a repeat is true, each opening is repeated for two
-		 * rounds; otherwise each game gets its own opening.
+		 * Each opening is played \a count times before
+		 * going to the next opening. The default is 1.
 		 */
-		void setOpeningRepetition(bool repeat);
+		void setOpeningRepetitions(int count = 1);
```

Comment:
This function is always called with an explicit count and I don't see why it would be called without it, so the default parameter value of 1 is not needed. The documentation ("The default is 1") should still remain.